### PR TITLE
Schema reference edits

### LINKED ISSUE #528: Review schema reference content

### PR DESCRIPTION
**Pull request by kathryn-ods**
# Overview

**What does this pull request do?** 
 
- updates the schema ref page 
- updates the schema browser page
- fixes issues with the urn resolving in sphinx docson and directives
- fixes other docson issues 

**How can a reviewer test or examine your changes?**

- Run "pip install -r requirements_test.txt". 
- If you aren't starting with a clean repo on disk, you will need to run 'git submodule update --remote'
- Build the docs
- Read through the browser and reference pages. Check all the links

Note that the browser widget has some 'hanging' AllOfs and AnyOfs, and some of the UI isn't great. It just needs to be good enough for now.

Also: on the reference page, there are inconsistencies with sub-headings: some are 'camelCase' and some 'Are Not'. We're not going to resolve that this time.

**Who is best placed to review it?**

@odscrachel 

Closes #528 #546 #682 #557

## Translations

- [x] New or edited strings appearing as a result of this PR will be picked up for translation
- [x] I've notified the translation coordinator of any new strings that will need
      translating. See: https://openownership.github.io/bods-dev-handbook/translations.html

## Documentation & Release

- [x] I've checked that the documentation builds without failing. See: https://github.com/openownership/data-standard#building-the-documentation-locally
- [x] I've thought about how and when this needs to be released. See:
      https://openownership.github.io/bods-dev-handbook/standard_releases.html      
- [x] I've updated the changelog, if necessary.
- [x] I've updated [reference.rst](https://standard.openownership.org/en/latest/schema/reference.html) to reflect any changes to schema structure or naming.
- [x] I've added or edited any other related documentation. See: https://github.com/openownership/bods-dev-handbook/blob/master/style_guide.md

### CODE REVIEW

**Comment by kd-ods** [comment]
@radix0000 - when I build the docs on this branch and look at the docson widget, I don't see the fixes that you showed me for #682 (i.e. being able to see the sub-properties under `entityType` and `publisher`).

**Comment by radix0000** [comment]
If you aren't starting with a clean repo on disk, you will need to run `git submodule update --remote`. docson is brought in as a sub module rather than a pip requirement. Let me know if you have any problems.

**Comment by kd-ods** [comment]
That worked - thanks!!

**Review by kd-ods — CHANGES_REQUESTED** [review]
I've reviewed this in case, @kathryn-ods, you're around on Wed any want to continue work on this page.

Otherwise, I'll incorporate the suggested changes on Wed and have @odscrachel review before merging.

File: docs/standard/reference.rst [hunk 1]
```diff
@@ -175,15 +280,52 @@ Source
...
+
+Statement
+---------
+
+.. json-value:: ../_build_schema/statement.json
+   :pointer: /$defs/Statement/description
+
+.. jsonschema:: ../_build_schema/statement.json
+   :pointer: /$defs/Statement
+   :collapse: source,annotations,publicationDetails
```
[the remark this answers is not in the document]
Comment (kd-ods) [reply docs/standard/reference.rst:300]: Left this in and added an external link to the obejct table.

**Comment by kd-ods** [comment]
@odscrachel - this is ready for your review.

**Review by odscrachel — APPROVED** [review]

**Review by kd-ods — APPROVED** [review]

**Pull request merged**